rca/account_management/templates/wagtailadmin/base.html

```diff
@@ -18,7 +18,7 @@
         A wagtail_hook adds some global JS and CSS to manage it's visibility in rca/people/wagtail_hooks.py
         {% endcomment %}
     </aside>
-    {% keyboard_shortcuts_dialog %}
+    {% comment %} {% keyboard_shortcuts_dialog %} {% endcomment %}
```

Comment:
Not sure what this is? Does it need to be here if it's commented out?

Reply:
It was raising an error when I access the admin so I commented it out.

This is needed though so I've added it back in.